Bug in E2E_EXISTING_CHART=true E2E_CERT_MANAGER_VERSION=1.14.2 make e2e-setup-certmanager

```sh
$ E2E_EXISTING_CHART=true E2E_CERT_MANAGER_VERSION=1.14.2 make e2e-setup-certmanager
/home/richard/projects/cert-manager/cert-manager/_bin/tools/kubectl apply --server-side -f _bin/scratch/gateway-api-v1.0.0.yaml > /dev/null
/home/richard/projects/cert-manager/cert-manager/_bin/tools/helm upgrade \
        --install \
        --create-namespace \
        --wait \
        --namespace cert-manager \
        --repo https://charts.jetstack.io \
        --version1.14.2 \
        --set crds.enabled=true \
        --set featureGates="AdditionalCertificateOutputFormats=true\,ExperimentalCertificateSigningRequestControllers=true\,ExperimentalGatewayAPISupport=true\,ServerSideApply=true\,LiteralCertificateSubject=true\,UseCertificateRequestBasicConstraints=true\,NameConstraints=true\,OtherNames=true" \
        --set "extraArgs={--kube-api-qps=9000,--kube-api-burst=9000,--concurrent-workers=200}" \
        --set webhook.featureGates="AdditionalCertificateOutputFormats=true\,LiteralCertificateSubject=true\,NameConstraints=true\,OtherNames=true" \
        --set "cainjector.extraArgs={--feature-gates=ServerSideApply=true}" \
        --set "dns01RecursiveNameservers=10.0.0.16:53" \
        --set "dns01RecursiveNameserversOnly=true" \
         \
         \
        cert-manager cert-manager >/dev/null
Error: unknown flag: --version1.14.2
make: *** [make/e2e-setup.mk:300: e2e-setup-certmanager] Error 1

```

/kind bug

--- a/make/e2e-setup.mk
+++ b/make/e2e-setup.mk
@@ -304,7 +304,7 @@ e2e-setup-certmanager: e2e-setup-gatewayapi $(E2E_SETUP_OPTION_DEPENDENCIES) $(b
 		--wait \
 		--namespace cert-manager \
 		--repo $(E2E_CERT_MANAGER_REPO) \
-		$(addprefix --version,$(E2E_CERT_MANAGER_VERSION)) \
+		$(addprefix --version=,$(E2E_CERT_MANAGER_VERSION)) \
 		--set crds.enabled=true \
 		--set featureGates="$(feature_gates_controller)" \
 		--set "extraArgs={--kube-api-qps=9000,--kube-api-burst=9000,--concurrent-workers=200,--enable-gateway-api}" \

